streaming/src/test/scala/io/gearpump/streaming/appmaster/AppMasterSpec.scala

```diff
@@ -93,8 +93,11 @@ class AppMasterSpec extends WordSpec with Matchers with BeforeAndAfterEach with
     mockMaster.reply(AppMasterRegistered(appId))
     mockMaster.expectMsg(15 seconds, GetAppData(appId, "DAG"))
     mockMaster.reply(GetAppDataResult("DAG", null))
-    mockMaster.expectMsgAllOf(15 seconds, GetAppData(appId, "startClock"), RequestResource(appId, ResourceRequest(Resource(4))))
+    mockMaster.expectMsg(15 seconds, GetAppData(appId, "startClock"))
```

Comment:
Will this cause UT failure ? I remember UT failed previously because `ResourceRequest` is received before `GetAppData`.